tutorials/howtos/responsive-images.md

```diff
@@ -3,7 +3,62 @@ title: "Responsive images"
 layout: tutorial
 ---
 
-The `apostrophe-images` widget provides a powerful way to select and display images on the site, including the ability to manually crop images. But sometimes, this isn't best for mobile-responsive site designs.
+The `apostrophe-images-widgets` module provides a powerful way to select and display images on the site, including the ability to manually crop images on the server or responsively on the client.
```

Comment:
I don't think the language about "manually on the server or responsively on the client" is clear. How about:

"Including the ability to manually crop images, or rely on CSS to carry out various automatic responsive cropping approaches."